tests/_test_utils/test_mssautoplot

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+"""
+
+    tests._test_utils.test_mssautoplot
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    This module provides pytest functions to test mslib.utils.mssautoplot
+
+    This file is part of MSS.
+
+    :copyright: Copyright 2022 Sreelakshmi Jayarajan
+    :copyright: Copyright 2022 by the MSS team, see AUTHORS.
+    :license: APACHE-2.0, see LICENSE for details.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+import os
+import mock
+import fnmatch
+import logging
+from click.testing import CliRunner
+from mslib.utils import autoplot
+from datetime import datetime, timedelta
+from tests.utils import wait_until_signal
+import mslib.msui.wms_control as wc
+
+PORTS = list(range(18000, 18500))
```

Comment:
please check that there is no collision with other PORTs in same range.

test_wms seems to have the same as this.

This is used by xdist (parallel tests) to not get in trouble starting many mswms server. 
And also to have tests always on fresh servers.